src/main/java/com/learntechfree/ltf/entity/Category.java

```diff
@@ -0,0 +1,27 @@
+package com.learntechfree.ltf.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
```

Comment:
Avoid importing unnecessary classes, import what we need.

Reply:
I hope this is automated by IDE 💯

Reply:
we can change the settings in IDE